src/fetchAPIData.js

```diff
@@ -30,6 +30,13 @@ export const apiRequest = {
       body: JSON.stringify(body),
     })
       .then(response => response)
-      .catch(error => console.log(error.message));
+    // .catch(error => displayError(error))
```

Comment:
is this supposed to be commented out?

Reply:
Yes, I just removed it. Handling the catch error on the invoked post request in index.js